system/hardware/force_cpu_rendering.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+
+SCRIPT_PATH="$(dirname "$BASH_SOURCE")"
+
+DEVICE_TYPE="$($SCRIPT_PATH/get_device_type.py)"
+
+# force CPU rendering on respective platforms
+if [ "$DEVICE_TYPE" == "tici" ]; then
+  export QT_QPA_PLATFORM=eglfs                                    # offscreen doesn't work with EGL/GLES
+  export LD_PRELOAD=/usr/lib/aarch64-linux-gnu/libEGL_mesa.so.0   # force loading mesa driver over adreno
```

Comment:
we probably want to check this into third_party/. can we also use the same stuff on PC?

Reply:
already exists on device